app/src/main/res/values/strings.xml

```diff
@@ -275,6 +277,7 @@
   <string name="password_shown_icon">Password shown icon</string>
   <string name="password_hidden_icon">Password hidden icon</string>
   <string name="enter_your_pin">Enter your PIN</string>
+  <string name="pin_password_toolbar">Enter PIN</string>
```

Comment:
Should this be ``pin_password_toolbar_title``? It's a bit unclear on what the "what" is corresponding to this string.

Reply:
Change it to `pin_password_toolbar_title`.